presentation/src/main/java/com/mashup/presentation/home/viewholder/SignalViewHolder.kt

```diff
@@ -0,0 +1,67 @@
+package com.mashup.presentation.home.viewholder
+
+import SimpleGapItemDecorator
+import android.app.Activity
+import android.os.Build
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.mashup.presentation.databinding.ListItemSignalBinding
+import com.mashup.presentation.home.adapter.SignalKeywordAdapter
+import com.mashup.presentation.home.model.SignalUiModel
+import eightbitlab.com.blurview.RenderEffectBlur
+
+
+class SignalViewHolder(
+    private val binding: ListItemSignalBinding,
+) : RecyclerView.ViewHolder(binding.root) {
+    private val keywordAdapter by lazy { SignalKeywordAdapter() }
+
+    init {
+        initView()
+    }
+
+    private fun initView() {
+        with(binding) {
+            val decorView = (root.context as Activity).window.decorView
+            val rootView = decorView.findViewById<ViewGroup>(android.R.id.content)
+
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
+                blurView.setupWith(rootView, RenderEffectBlur())
+                    .setBlurRadius(BLUR_RADIUS)
+            }
+            blurView.clipToOutline = true
+
+            keywordRecyclerView.adapter = keywordAdapter
```

Comment:
개인적인 생각으로는 키워드 RecyclerView가 3개의 키워드 + 갯수에 해당하는 itemView를 모든 시그널 RecyclerView의 itemView에서 보여야 하는데, 이게 매번 스크롤 될때마다 키워드 리사이클러뷰 아이템이 pool까지도 저장될 것 같은데, 캐시 사이즈를 늘려보는건 어떨까?

```suggestion
            keywordRecyclerView.setItemViewCachedSize(5)
            keywordRecyclerView.adapter = keywordAdapter
```